src/Components/Alert/Components/Filters/BaseAlertFilter.tsx

```diff
@@ -0,0 +1,73 @@
+import { Checkbox, Column, GridColumns, Spacer, Text } from "@artsy/palette"
+import { FC } from "react"
+import { useAlertContext } from "Components/Alert/Hooks/useAlertContext"
+import { ShowMore } from "Components/Alert/Components/Filters/ShowMore"
+import { SearchCriteriaAttributeKeys } from "Components/SavedSearchAlert/types"
+
+interface BaseAlertFilterProps {
+  criteriaKey: SearchCriteriaAttributeKeys
+  description?: string
+  expanded?: boolean
```

Comment:
**suggestion**: Based on conversations in Slack, perhaps we defer the addition of this `expanded` prop until we need it?

Reply:
Hmm, I would be better leaving it, given that we will need the "Show More" logic when implementing other filters, and removing the code would be more work than leaving it.

But yeah, you're right. I would not have added this logic if if I had known about the new requirements before working on the ticket.

But I'm happy to remove it if you feel strongly about it.

Reply:
Sure, let's leave it in for now — we can revisit later.

I wonder how many of those filters will utilize "Show more" given the questions around selected filters below the fold or repositioning those selected filters to the top of the list.

We also need to figure out whether we can port this proposed pattern for app over to Desktop Web — which would also obviate the need for a "Show more" implementation.

![Screenshot 2023-11-07 at 14 39 19](https://github.com/artsy/force/assets/123595/12a0c73f-872f-4c09-ad38-c501c7186daf)

Reply:
Yeah, it might be that we won't need the "Show More" pattern at all in the end.